.github/workflows/EngineBuild.yml

```diff
@@ -0,0 +1,48 @@
+name: Engine
+
+on:
+  pull_request: {}
+  push: { branches: [master] }
+
+env:
+  UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+  UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+  UNITY_SERIAL: ${{ secrets.UNITY_SERIAL }}
+
+jobs:
+  buildForSomePlatforms:
+    name: Build for ${{ matrix.targetPlatform }} on version ${{ matrix.unityVersion }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        projectPath:
+          - engine/CoreEngine
+        unityVersion:
+          - 2019.3.14f1
```

Comment:
This needs to be a environment variable in the repo that we can change as the modern unity gets updated. If you don't want to target the beta unity version this makes the most sense.

Reply:
Now updated to latest ✔️